Week2/homework/maartjes-work.js

```diff
@@ -45,21 +45,32 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
-
-// eslint-disable-next-line no-unused-vars
-const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
-
+// Start: Adham's code
+const maartjesDurationsHours = []; /* Collect our data in one array */
+maartjesTasks.forEach(element => {
+  const durationMinutes = element.duration;
+  const durationHours = durationMinutes / 60;
+  if (durationHours >= 2) {
+    maartjesDurationsHours.push(durationHours);
+  }
+});
+// console.log(maartjesDurationsHours); /* Array of durations in hours, just if the duration is more than 2 hours */
+const reducingProcess = (totalToLeft, right) => totalToLeft + right;
```

Comment:
Hi Adham - similar to my comment in the doubling odd numbers exercise, I think we can combine these two operations in a single statement. Something like:

```
const maartjesTotalDuration = maartjesDurationHours.reduce((total, hours) => total + hours);
```

It has got similar advantages as I have mentioned in that comment 🙂

Reply:
Yes , I have to consider that seriously . 
Thanks